### PR TITLE
[13.0][MIG] account_move_budget

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @ForgeFlow

### CODE REVIEW

File: account_move_budget/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1,3 +1,3 @@
-* Eficent Business and IT Consulting Services, S.L. (https://www.eficent.com)
-  * Hector Villarreal <hector.villarreal@eficent.com>
-  * Jordi Ballester Alomar <jordi.ballester@eficent.com>
+* ForgeFlow, S.L. (https://www.eficent.com)
```
Comment (nikul-serpentcs) [inline account_move_budget/readme/CONTRIBUTORS.rst]:
```suggestion
* ForgeFlow, S.L. (http://www.forgeflow.com)
```

File: account_move_budget/models/account_move_budget_line.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    def _compute_store_balance(self):
+        for line in self:
+            line.balance = line.debit - line.credit
+
+    @api.constrains("date")
+    def _constraint_date(self):
+        for rec in self:
+            if rec.budget_id.date_from > rec.date or rec.budget_id.date_to < rec.date:
+                raise ValidationError(
+                    _("The date must be within the " "budget period.")
```
Comment (guewen) [inline account_move_budget/models/account_move_budget_line.py]:
```suggestion
                    _("The date must be within the budget period.")
```
Glitch introduced by black.

**Review by guewen — APPROVED** [review]

File: account_move_budget/__manifest__.py [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
 {
     "name": "Account Move Budget",
     "summary": "Create Accounting Budgets",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Accounting & Finance",
     "website": "https://github.com/OCA/account-financial-tools",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline account_move_budget/__manifest__.py:10]:
```suggestion
    "author": "ForgeFlow, Odoo Community Association (OCA)",
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

You may squash last commit.

**Comment by simahawk** [comment]
@HviorForgeFlow can you rebase, run pre-commit again, squash... etc? Thanks :)

**Pull request merged**